RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -799,14 +818,24 @@ boolean L12_filthworms()
 		use(1, $item[Filthworm Royal Guard Scent Gland]);
 	}
 
+	//if we can kill the queen we don't care about gland drops anymore. kill her and finish this
 	if(have_effect($effect[Filthworm Guard Stench]) > 0)
 	{
-		handleFamiliar("meat");
```

Comment:
Queen has 1k meat drop so if this isn't handled elsewhere you might want to revert it.

Reply:
change made